ci/jenkins/test.sh

```diff
@@ -49,9 +49,12 @@ NETWORKPOLICY_SKIP="should allow egress access to server in CIDR block|should en
 
 CONTROL_PLANE_NODE_ROLE="master|control-plane"
 
-CLEAN_STALE_IMAGES="docker system prune --force --all --filter until=48h"
+CLEAN_STALE_IMAGES="docker system prune --force --all --filter until=4h"
 CLEAN_STALE_IMAGES_CONTAINERD="crictl rmi --prune"
 
+PRINT_DOCKER_STATUS="docker system df -v"
+PRINT_CONTAINERD_STATUS="crictl ps --state Exited"
```

Comment:
Question: Is this the same as `"docker system df -v"`, it looks like print exited containers instead of containerd system level info.

Reply:
If it's the same purpose, should we add it to test-mc.sh? it also supports containerd.

Reply:
> Question: Is this the same as `"docker system df -v"`, it looks like print exited containers instead of containerd system level info.

Yes, since containerd doesn't support "system" command, we just show exited container status here.

> If it's the same purpose, should we add it to test-mc.sh? it also supports containerd.

Sure, and I also found test-mc.sh still use docker cleanup function for containerd testbed, I have updated the script.